packages/mattermost-redux/src/selectors/entities/posts.ts

```diff
@@ -351,19 +353,26 @@ export function makeGetPostsForThread(): (state: GlobalState, rootId: string) =>
     return createIdsSelector(
         'makeGetPostsForThread',
         getAllPosts,
+        getCurrentUser,
         (state: GlobalState, rootId: string) => state.entities.posts.postsInThread[rootId],
         (state: GlobalState, rootId: string) => state.entities.posts.posts[rootId],
-        (posts, postsForThread, rootPost) => {
+        getMyPreferences,
```

Comment:
FYI we have a selector that returns true/false based on preferences [`shouldShowJoinLeaveMessages`](https://github.com/mattermost/mattermost-webapp/blob/master/packages/mattermost-redux/src/utils/post_list.ts#L28)

Reply:
Done thanks

Reply:
`getMyPreferences` and `myPreferences` should not be needed now, right? how come eslint didn't complain on `myPreferences` not being used 🤔 , am I missing something?